resources/js/features/games/components/GameSetRequests/GameSetRequestsRoot.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FC } from 'react';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { GameBreadcrumbs } from '@/common/components/GameBreadcrumbs';
+import { GameHeading } from '@/common/components/GameHeading';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { PatreonSupporterItem } from '@/features/patreon-supporters/components/PatreonSupporterItem';
+
+export const GameSetRequestsRoot: FC = () => {
+  const { game, deferredRequestors, initialRequestors, totalCount } =
+    usePageProps<App.Community.Data.GameSetRequestsPageProps>();
+  const { t } = useTranslation();
+
+  // Combine initial and deferred requestors once deferred props finish loading.
+  const allRequestors = useMemo(() => {
+    const requestors = [...initialRequestors];
+    if (deferredRequestors) {
+      requestors.push(...deferredRequestors);
+    }
+
+    return requestors;
+  }, [initialRequestors, deferredRequestors]);
+
+  return (
+    <div className="flex flex-col gap-5">
+      <GameBreadcrumbs game={game} system={game.system} t_currentPageLabel={t('Set Requests')} />
+      <GameHeading game={game} wrapperClassName="!mb-1">
+        {t('Set Requests')}
+      </GameHeading>
+
+      <div>{'A set for this game has been requested by the following users:'}</div>
+
+      <div>
```

Comment:
Recommend adding an empty list special case:

<img width="455" height="234" alt="Image" src="https://github.com/user-attachments/assets/14df11bc-5bc9-405e-b365-7ca9e10e25af" />

Reply:
Maybe changing the text to something like "This game currently has no requests. Could you be the first?"